static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -201,6 +202,43 @@ function CrumbErrorIssue({frame}: {frame: ErrorFrame}) {
   );
 }
 
+function HTMLTree({description, frame}: {description: string; frame: ClickFrame}) {
+  const location = useLocation();
+  const organization = useOrganization();
+
+  const componentName =
+    isClickFrame(frame) && frame.data.node?.attributes['data-sentry-component'];
+  const lastComponentIndex =
+    description.lastIndexOf('>') === -1 ? 0 : description.lastIndexOf('>') + 2;
+
+  return (
+    <Description title={description} showOnlyOnOverflow isHoverable>
```

Comment:
what happens if the there's a react component name & the description here overflows? will there be two layers of tooltips :o

Reply:
I changed it so that there's 2 layers of tooltips :D